### PR TITLE
fix doc "Artifact transform selection and execution"

### PR DESCRIPTION
**Pull request by reinsch82**
fix broken example for kotlin and groovy

Resolves: #22962

### CODE REVIEW

**Comment by eskatos** [comment]
@bot-gradle help

File: subprojects/docs/src/docs/userguide/dep-man/04-modeling-features/artifact_transforms.adoc [hunk 1]
```diff
@@ -95,6 +95,7 @@ include::sample[dir="snippets/dependencyManagement/artifactTransforms-minify/gro
 <2> All JAR files are not minified
 <3> Request `minified=true` on all resolvable configurations
 <4> Add the dependencies which will be transformed
+<5> Create Task to execute the artifact transformation
```
Comment (ljacomet) [inline subprojects/docs/src/docs/userguide/dep-man/04-modeling-features/artifact_transforms.adoc:98]: 💄 The rest of the bullet points use "Add" as the verb, I would do the same here for consistency.

**Review by ljacomet — APPROVED** [review]
LGTM!

Only a minor style remark

**Comment by ljacomet** [comment]
In order to have this in 8.0, it will need to be merged to `release` instead of the main branch.

File: subprojects/docs/src/docs/userguide/dep-man/04-modeling-features/artifact_transforms.adoc [hunk 2]
```diff
@@ -95,6 +95,7 @@ include::sample[dir="snippets/dependencyManagement/artifactTransforms-minify/gro
 <2> All JAR files are not minified
 <3> Request `minified=true` on all resolvable configurations
 <4> Add the dependencies which will be transformed
+<5> Add task to execute the artifact transformation
```
Comment (big-guy) [inline subprojects/docs/src/docs/userguide/dep-man/04-modeling-features/artifact_transforms.adoc:98]: "execute" here makes it sound like resolveRuntimeClasspath is doing the work of the artifact transform.

What about...
```suggestion
<5> Add task that requires the transformed artifacts
```

**Review by ljacomet — APPROVED** [review]

**Comment by ljacomet** [comment]
@bot-gradle test and merge

**Pull request merged**